qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/OpenConceptLabPage.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+
+public class OpenConceptLabPage extends Page {
+    private static final By SETUP_SUBSCRIPTION_BUTTON = By.cssSelector("#body-wrapper > home > div:nth-child(2) > div:nth-child(2) > a");
+    public OpenConceptLabPage(Page page) {
+        super(page);
+    }
+    
+    @Override
+    public String getPageUrl() {
+        return "/owa/openconceptlab/index.html#/";
```

Comment:
i would return url definition , to look more readable.
```suggestion
        return  URL;
```